### ISSUE TITLE
The different results obtained from the paper are for N2

### ISSUE THREAD

**Issue by sunny-viv** [google-deepmind/ferminet#94]
Hello, I found different results in reproducing the ground state energy of N2 molecule in the paper. The result in the literature is − 109.5388, but the result I run 10,000 times is about − 108.8. I want to know what configuration is used in the original literature ? For example, bath _ size, determinats, optim.lr.rate and so on. I use adam as the optimizer. Here is my input configs used. thank you!

from ferminet import base_config
from ferminet.utils import system
import ml_collections
import numpy as np
import os

def get_config() -> ml_collections.ConfigDict:
    """Returns DMC-optimized config for running N2 with FermiNet on CPU."""
    cfg = base_config.default()
  
    bond_length = 2.074
    
    cfg.system.molecule = [
        system.Atom(symbol='N', coords=(0.0, 0.0, -bond_length/2)),
        system.Atom(symbol='N', coords=(0.0, 0.0, bond_length/2)),
    ]
    
    cfg.system.electrons = (7, 7)
    cfg.batch_size = 256 
    cfg.pretrain.iterations = 2000 
    cfg.optim.iterations = 10000
    
    cfg.optim.lr.rate = 1e-4
    cfg.optim.lr.decay_steps = 5000  
    cfg.optim.lr.decay_rate = 0.95  
    cfg.optim.lr.min = 1e-5  
    
    cfg.optim.optimizer = 'adam
    cfg.mcmc.burn_in = 1000 
    cfg.mcmc.steps = 50  
    cfg.mcmc.move_width = 0.02  
    
    cfg.network.determinants = 16  
    cfg.network.hidden_dims = ((128, 128, 128), (128, 128))
    cfg.network.use_last_layer = True
   
    cfg.optim.clip_el = True 
    cfg.optim.clip_el_sigma_scale = 5.0 

    cfg.log.save_frequency = 720  # Save checkpoint every hour
    cfg.log.save_path = 'logs/n2'  # Save directory
    
    # 8. JAX CPU optimizations
    cfg.system.jax_config = {
        'jax_enable_x64': True,  # Use double precision
        'jax_disable_jit': False,  # Keep JIT enabled
        'jax_platform_name': 'cpu',  # Force CPU platform
    }
    
    # 9. Additional precision settings
    cfg.system.dtype = 'float64'  # Use double precision
    
    # Create log directory if it doesn't exist
    os.makedirs(cfg.log.save_path, exist_ok=True)
    
    return cfg

**Comment by jsspencer** [comment]
Please see Table IV and Table V in the supplementary material of [Phys Rev Research 2, 033429](https://doi.org/10.1103/PhysRevResearch.2.033429). In particular, we ran with a much later batch size (4096), for more iterations (at least 200k) and used KFAC (Fig 3 justifies this). The learning rate can be increased to ~0.05, as the JAX version uses a different normalization of the gradients. Note after training, a separate inference run (without parameter updates) is required to obtain an accurate estimate of the energy (see appendix A1).

For N2, single precision should be fine. I haven't run N2 on a CPU though, this is likely to be very slow.

**Comment by sunny-viv** [comment]
thank you so much!
